src/utils/compat/realpath.c

```diff
@@ -0,0 +1,225 @@
+/*	$OpenBSD: realpath.c,v 1.13 2005/08/08 08:05:37 espie Exp $ */
+/*
+ * Copyright (c) 2003 Constantin S. Svintsoff <kostik@iclub.nsu.ru>
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
```

Comment:
Do we fulfill point number 2 here? Does dlite reproduce this copyright notice?

Reply:
Very good point. We don't do that at the moment. Is the correct way to add a dlite_get_license() function that returns  a pointer to a string with all the copyright notes? 

To make this easy to maintain, I guess that we should prepare one text file with all licenses (should it be LICENSE.txt or a separate file?) and generate the dlite_get_license() function during the build process.

Reply:
The above is now implemented. In C we now have `dlite_get_license()` while Python has `dlite.get_license()`.